torchvision/datasets/hmdb51.py

```diff
@@ -50,7 +50,8 @@ class HMDB51(VisionDataset):
     }
 
     def __init__(self, root, annotation_path, frames_per_clip, step_between_clips=1,
-                 fold=1, train=True, transform=None):
+                 frame_rate=None, precomputed_metadata=None, precomputed_metadata_filepath=None,
```

Comment:
I would be ok exposing the `_precomputed_metadata` as a private argument in the constructor for now, and also expose the `frame_rate`, but I think that the `precomputed_metadata_filepath` and `save_metadata_filepath` should live outside of the dataset and be handled by the training code.

Reply:
I agree. @fmassa